Package.swift

```diff
@@ -33,7 +33,7 @@ let package = Package(
   ],
 
   dependencies: [
-    .package(url: "https://github.com/apple/swift-syntax.git", from: "509.0.0"),
+    .package(url: "https://github.com/apple/swift-syntax.git", branch: "main"),
```

Comment:
This is temporary for the sake of CI; I'll switch it back after it passes.

Reply:
For CI, does this change also need to be applied (temporarily) to `Package@swift-6.0.swift`?

Reply:
🤔 possibly. Will do that too.